docs/specs/templates.md

```diff
@@ -126,3 +127,22 @@ One-time-password message. Used for authentication and user verification.
     - `Host`: Host of authgear, usually used for [Web OTP](https://web.dev/web-otp/) API.
     - `Origin`: The origin page of the OTP message, can be `signup`/`login`/`settings`.
     - `Operation`: The operation triggering the OTP message, can be `primary_auth`/`secondary_auth`/`verify`.
+
+### `oob_magic_link`
+
+Magic link. Used for authentication and user verification.
+
+- Template types:
+    - `authenticate_primary_oob_magic_link.txt` / `authenticate_secondary_oob_magic_link.txt`
+    - `authenticate_primary_oob_magic_link.html` / `authenticate_secondary_oob_magic_link.html`
+- Context:
+    - `AppName`: the display name of the app.
+    - `Email`: The recipient email of the message.
+    - `Phone`: The recipient phone number of the message.
+    - `LoginID`: The login ID of the identity
+        - `LoginID.Key`: Login ID key
+        - `LoginID.Value`: Login ID
+    - `Link`: The magic link.
+    - `Host`: Host of authgear.
+    - `Origin`: The origin page of the magic link, can be `signup`/`login`/`settings`.
```

Comment:
Refer to pkg/lib/authn/otp/template.go MessageTemplateContext to see what variables are actually available. Remove anything that is not implemented.